packages/camp/src/NumberInput/NumberInput.tsx

```diff
@@ -48,6 +55,11 @@ export const NumberInput = forwardRef<NumberInputProps, 'input'>(
       isPrefilled,
     })
 
+    const mergedInputStyles = useMemo(
+      () => mergeThemeOverride(styles.field, inputStyles),
+      [inputStyles, styles.field],
+    )
```

Comment:
might not need to use `mergeThemeOverride` and just assign `inputStyles` directly to input's `sx` prop. Chakra UI has a priority list for styles (`_css` < `sx`). Can you try and see whether just using sx with inputStyles is enough?

Reply:
you're right, it works. Just updated